### ISSUE TITLE
Restrict default search view for logged-in users to eligible studies

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1128]
# TL;DR

Rather than users actively selecting from a drop-down to see studies for Child1 or Child2, can we have the default to show studies they are eligible for?  On the UI side, this might look like 'tag' filters that can be added/removed by the family: 

Default tags: 

x[Child 1] x[Child 2] x[Child 3]

Where you can click X to remove a tag. If no child has been selected, then show all studies. 

# Narrative

As a family, I want to encounter studies I can actually take when I click on them.  When I'm logged in, the first study I click on I may not be eligible for, because I'm shown all the listed studies.

# Acceptance Criteria

- Make sure that child/tag selection interacts appropriately with the other filters! (Studies we've already done, studies on/off lookit, tabs for external/internal studies)
